.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Pull Request
+---
+# What are the relevant tickets?
+<!--- If it fixes an open issue, please link to the issue here. -->
+<!--- Closes # --->
+<!--- Fixes # --->
+
+## Description (What does it do?)
+<!--- Describe your changes in detail -->
+
+## How can this be tested?
+<!--- Please describe in detail how your changes have been tested. -->
+<!--- Include details of your testing environment, any set-up required --->
+<!--- (e.g. data entry required for validation) and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+<!--- Please also include instructions for how your reviewer can validate your changes. --->
+
+
+## Screenshots (if appropriate):
```

Comment:
This is not relevant for some projects (like devops). Perhaps add it as a comment under "Description (what does it do?" above ?